css/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(243, 243, 243);
+}
+
+form {
+  padding: 0 1.5rem;
+}
+
+.column-full {
+  padding: 0;
+}
+
+h1 {
+  background-color: rgb(93, 25, 110);
```

Comment:
instead of setting `background-color` on the `h1`, set it to the `header` element. That way it'll go across the entire width of the page and will fix the padding issue I was describing in the last review. Because it's set on the `header` it will override the setting you put on `body`.

Technically the padding is still there for the `body` content but since you set the `background-color` on the `body` element in this update it'll look cleaner across the board.